_tests_/index.scss

```diff
@@ -1,87 +1,27 @@
 @use "sass:color";
-@import "import.css";
-@use "use" as *;
+
+@mixin use-mixin {
+    color: red;
+}
 
 $var-name: "prop" !default;
 $bg: rgba(#ddd, 0.3) !default;
 
-.order-example {
+.scss-example {
     @include use-mixin;
-
-    /* var's/special setters */
     --color-theme: hsl(270 50% 40%);
     --#{$var-name}: "value";
-    content: attr();
-    counter-reset: section;
-    counter-increment: section;
-    quotes: "“" "”" "‘" "’";
-    box-sizing: border-box;
-
-    /* Positioning */
-    z-index: 1;
```

Comment:
Idem voor het vorige punt, zijn dit dingen die ergens anders getackled worden en daardoor hier niet meer nodig is?